libathemecore/ctcp-common.c

```diff
@@ -67,7 +67,7 @@ ctcp_about_handler(struct sourceinfo *si, char *cmd, char *args)
 void
 common_ctcp_init(void)
 {
-	ctcptree = mowgli_patricia_create(noopcanon);
+	ctcptree = mowgli_patricia_create(irccasecanon	);
```

Comment:
This should use `strcasecanon` in case the configured match mapping is not ASCII.